### ISSUE TITLE
Why the "&nbsp;" is replaced by a whitespace?

### ISSUE THREAD

**Issue by AlkaTale** [microsoft/roosterjs#430]
I'm testing Talkback(a screen reader) on an app using roosterjs and find a bug. When typing the first character of a word, Talkback should announce the character, but it announces "replaced". 

This is the screenshot of this bug:
<img width="387" alt="WX20200616-215040@2x" src="https://user-images.githubusercontent.com/29978667/84784827-55e78d80-b01d-11ea-998b-786ec0a18d49.png">

I have found the cause: the `&nbsp;` at the end is replaced by a whitespace when typing next character. So Talkback announces "replaced". Why does roosterjs replace the `&nbsp;`?

These are the editor dom before and after the replacement:
<img width="664" alt="WX20200616-220746@2x" src="https://user-images.githubusercontent.com/29978667/84785509-25ecba00-b01e-11ea-845e-58289d39b21e.png">
<img width="671" alt="WX20200616-220812@2x" src="https://user-images.githubusercontent.com/29978667/84785572-37ce5d00-b01e-11ea-948e-33cd7b335366.png">

**Comment by JiuqingSong** [comment]
If you are using Chrome, what you have seen is the default behavior of Chrome. You can try open a very simple HTML page in Chrome like this:
```html
<html>
<body>
<div contenteditable style="width:100px; height: 100px"></div>
</body>
</html>
```
And try type in the box, you will get exact same behavior. See there isn't any javascript code but Chrome just does this.

**Comment by AlkaTale** [comment]
> ```
> <html>
> <body>
> <div contenteditable style="width:100px; height: 100px"></div>
> </body>
> </html>
> ```

Yes, I tried and got the same behavior. Thanks for your reply!
